book/files-modules-and-programs/README.md

```diff
@@ -648,14 +678,19 @@ terseness and explicitness—the more modules you open, the fewer module
 qualifications you need, and the harder it is to look at an identifier and
 figure out where it comes from.
 
-Here's some general advice on how to deal with `open`s: [local opens]{.idx}
+Here's some general advice on how to deal with `open`s:
 
-- Opening modules at the toplevel of a module should be done quite sparingly,
-  and generally only with modules that have been specifically designed to be
-  opened, like `Base` or `Option.Monad_infix`.
+### Open modules rarely
 
-- If you do need to do an open, it's better to do a *local open*. There are
-  two syntaxes for local opens. For example, you can write:
+Opening modules at the toplevel of a module should be done quite
```

Comment:
The term 'toplevel' here might be slightly confusing, given it's also the term for the interactive REPL.

Reply:
Good point. Fixed that with some slightly wider rewording.